install.sh

```diff
@@ -17,27 +17,44 @@ usage() {
 Installs code-server for Linux, macOS and FreeBSD.
 It tries to use the system package manager if possible.
 After successful installation it explains how to start using code-server.
+
+Pass --start to startup code-server immediately, print the URL it can be
+accessed at and the initial password. Then the script will tail code-server's logs.
+
+Pass in user@host to install code-server on user@host over ssh.
+Pass --start to forward the code-server port so that you can immediately access it.
+
+If you rerun the script, code-server will be updated only as required.
 ${not_curl_usage-}
 Usage:
 
-  $arg0 [--dry-run] [--version X.X.X] [--method detect] [--prefix ~/.local]
+  $arg0 [--dry-run] [--version X.X.X] [--method detect] [--prefix ~/.local] [--start] [user@host]
```

Comment:
In `--method detect` the `detect` keyword is variable.

In `--start` we have `user@host` separated. Is this intentional?

Reply:
Yes, `--start` can be used without `user@host`.

Reply:
Ah I understand.